src/Quadrature/gaussquad_tri_table.jl

```diff
@@ -72,6 +72,584 @@ function _get_gauss_tridata(n::Int)
+    0.1882035356190327 0.6235929287619346 0.0796477389272102 / 2.0
+    0.0447295133944527 0.9105409732110945 0.0255776756586980 / 2.0
+    0.0206349616025249 0.4896825191987376 0.0313347002271392 / 2.0
+    0.1258208170141270 0.4370895914929365 0.0778275410047742 / 2.0
+    0.6235929287619346 0.1882035356190327 0.0796477389272102 / 2.0
+    0.9105409732110945 0.0447295133944527 0.0255776756586980 / 2.0
+    0.0368384120547363 0.2219629891607658 0.0432835393772893 / 2.0
+    0.7411985987844980 0.0368384120547363 0.0432835393772893 / 2.0
+    0.2219629891607658 0.7411985987844980 0.0432835393772893 / 2.0
+    0.2219629891607658 0.0368384120547363 0.0432835393772893 / 2.0
+    0.7411985987844980 0.2219629891607658 0.0432835393772893 / 2.0
+    0.0368384120547363 0.7411985987844980 0.0432835393772893 / 2.0];
+    elseif n == 10
+    xw=[0.3333333333333333 0.3333333333333333 0.0908179903827535 / 2.0
+    0.4855776333836573 0.4855776333836573 0.0367259577564668 / 2.0
+    0.1094815754850370 0.1094815754850370 0.0453210594355280 / 2.0
+    0.4855776333836573 0.0288447332326853 0.0367259577564668 / 2.0
+    0.1094815754850370 0.7810368490299259 0.0453210594355280 / 2.0
+    0.0288447332326853 0.4855776333836573 0.0367259577564668 / 2.0
+    0.7810368490299259 0.1094815754850370 0.0453210594355280 / 2.0
+    0.1417072194148800 0.3079398387641210 0.0727579168454201 / 2.0
+    0.0250035347626864 0.2466725606399027 0.0283272425310575 / 2.0
+    0.0095408154002995 0.0668032510122003 0.0094216669637328 / 2.0
+    0.5503529418209990 0.1417072194148800 0.0727579168454201 / 2.0
+    0.7283239045974108 0.0250035347626864 0.0283272425310575 / 2.0
+    0.9236559335875002 0.0095408154002995 0.0094216669637328 / 2.0
+    0.3079398387641210 0.5503529418209990 0.0727579168454201 / 2.0
+    0.2466725606399027 0.7283239045974108 0.0283272425310575 / 2.0
+    0.0668032510122003 0.9236559335875002 0.0094216669637328 / 2.0
+    0.3079398387641210 0.1417072194148800 0.0727579168454201 / 2.0
+    0.2466725606399027 0.0250035347626864 0.0283272425310575 / 2.0
+    0.0668032510122003 0.0095408154002995 0.0094216669637328 / 2.0
+    0.5503529418209990 0.3079398387641210 0.0727579168454201 / 2.0
+    0.7283239045974108 0.2466725606399027 0.0283272425310575 / 2.0
+    0.9236559335875002 0.0668032510122003 0.0094216669637328 / 2.0
+    0.1417072194148800 0.5503529418209990 0.0727579168454201 / 2.0
+    0.0250035347626864 0.7283239045974108 0.0283272425310575 / 2.0
+    0.0095408154002995 0.9236559335875002 0.0094216669637328 / 2.0];
+    elseif n == 11
+    xw=[0.5346110482707582 0.5346110482707582 0.0009270063289607 / 2.0
+    0.3989693029658552 0.3989693029658552 0.0771495349148131 / 2.0
+    0.2033099004312824 0.2033099004312824 0.0593229773807741 / 2.0
+    0.1193509122825813 0.1193509122825813 0.0361845405034180 / 2.0
+    0.0323649481112759 0.0323649481112759 0.0136597310026779 / 2.0
+    0.5346110482707582 -0.0692220965415165 0.0009270063289607 / 2.0
```

Comment:
Some points like this are located outside the reference triangle, is that correct? I have never seen that before.